### PR TITLE
default project formats to filtered project name

### PR DESCRIPTION
**Pull request by HughePaul**

### CODE REVIEW

File: config/default.yaml [hunk 1]
```diff
@@ -21,7 +22,7 @@ config:
     #    credentials: username:password
         baseUrl: https://jenkins.example.com
         saveCache: true
-        projectFormat: build-{project:alpha}-snapshot.yml
+        projectFormat: '{project:alpha}.yml'
```
Comment (PauloPortugal) [inline config/default.yaml]: :+1:

File: config/default.yaml [hunk 2]
```diff
@@ -10,9 +10,10 @@ config:
     #    credentials: username:password
         baseUrl: https://jenkins.example.com
         saveCache: true
-        projectFormat: build-{project:alpha}-snapshot
+        projectFormat: '{project:alpha}'
```
Comment (PauloPortugal) [inline config/default.yaml]: :+1:

File: config/default.yaml [hunk 3]
```diff
@@ -10,9 +10,10 @@ config:
...
         # promotion environments
         environments:
+            prd_aws: prd
```
Comment (PauloPortugal) [inline config/default.yaml]: why not `prd` instead of `prd_aws`?

File: config/default.yaml (hunk 1 above)
Comment (PauloPortugal) [reply config/default.yaml]: Why do you need the single quotes?

File: config/default.yaml (hunk 3 above)
Comment (HughePaul) [reply config/default.yaml]: this maps prd_aws environment for old prd promotions on an old jenkins onto the prd tag

File: config/default.yaml (hunk 1 above)
Comment (HughePaul) [reply config/default.yaml]: without them yaml thinks this is an object { "project": "alpha" }

File: config/default.yaml (hunk 3 above)
Comment (PauloPortugal) [reply config/default.yaml]: ;)

File: config/default.yaml (hunk 1 above)
Comment (PauloPortugal) [reply config/default.yaml]: got it!

**Comment by PauloPortugal** [comment]
All looks good to me!

**Pull request merged**